premarum-web-client/pages/landing/index.tsx

```diff
@@ -13,6 +13,15 @@ export default function Landing() {
     const theme = useTheme()
     const matches = useMediaQuery(theme.breakpoints.down('sm'), {noSsr:true}); // it reloads the iframe everytime it fetches this query (not intended)
     const [animation, setAnimation] = React.useState(false);
+    const [tabType, setTabType] = useState<"fullWidth" | "standard">('fullWidth')
```

Comment:
Gives priority to mobile devices. Once in client it can decide to render the desktop tabs instead